lsp-diagnostics.el

```diff
@@ -214,6 +224,16 @@ See https://github.com/emacs-lsp/lsp-mode."
   (add-hook 'lsp-diagnostics-updated-hook #'lsp-diagnostics--flycheck-report nil t)
   (add-hook 'lsp-managed-mode-hook #'lsp-diagnostics--flycheck-report nil t))
 
+(defun lsp-diagnostics-flycheck-disable (&rest _)
```

Comment:
Is there a reason for it to take `&rest` arguments?

Reply:
no, just becayse -enable had it

Reply:
i removed it